pkg/utils/db.go

```diff
@@ -119,7 +119,10 @@ func EstimateRowsCount(db *sql.DB, schemaName string, tableName string) (int64,
 }
 
 func IsColumnString(columnType *sql.ColumnType) bool {
-	return strings.Contains(columnType.DatabaseTypeName(), "TEXT") || strings.Contains(columnType.DatabaseTypeName(), "CHAR")
+	log.Infof("IsColumnString: columnType: %v", columnType.DatabaseTypeName())
```

Comment:
debug? this function is called quite frequently I think

Reply:
changed